packages/node/src/index.ts

```diff
@@ -460,6 +460,7 @@ export async function startDevServer(
     entrypoint.endsWith('.mjs') ||
     (pkg.type === 'module' && entrypoint.endsWith('.js'));
 
+  console.log({ isEsm, entrypoint });
```

Comment:
arbitrary change to `@vercel/node` to make sure we run the tests...I'll remove after CI completes

Reply:
Ok CI is passing in 2edecba ✅ 

I reverted this in 433c8db 👍